### PR TITLE
Feature/seperate market context test

### PR DESCRIPTION
**Pull request by TzuHanLiang**
### CHECK LIST

- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  0
- new loop: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/market_section/market_section.tsx [hunk 1]
```diff
@@ -32,20 +33,20 @@ const MarketSection = (props: IMarketSectionProps) => {
...
-      <div className="ml-5">{displayedTickerHeader}</div>
-
-      <div className="mx-auto max-w-1920px container">{displayedTradingView}</div>
-
+      <TickerProvider>
+        <div className="ml-5">
+          <TradingHeader />
+        </div>
+        <div className="mx-auto max-w-1920px container">
+          {props?.hideTradingView ? null : <TradingView />}
```
Comment (arealclimber) [inline src/components/market_section/market_section.tsx:47]: 建議 return 裡面不要寫邏輯

**Review by Luphia — APPROVED** [review]
- no bug found

**Comment by TzuHanLiang** [comment]
- 新增對 workerContext 的調整，原本的寫法會在多個請求同時進行時，只有最後一個請求可以得到正確的回傳，其餘的會出錯

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**